### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 42

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#230] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255371-07864cef-f06c-4412-9d62-a66142e61f20.png)

**Comment by lord-marcel** [comment]
![es42](https://user-images.githubusercontent.com/36630876/152648878-56f560bb-fd23-48cf-a9d1-8e222ee103af.png)

**Comment by i0nut02** [comment]
nella terza parte dell'esercizio stai considerando anche i casi in cui dai più punti ad una singola domanda, ad esempio
esiste il caso in cui dai 5 punti alla prima domanda che vale solo un punto al massimo, quindi dovresti considerare tutti i casi - i casi in cui assegni almeno 2 punti a un elemento , e il caso in cui 2 risposte hanno 2 punti 

> ![es42](https://user-images.githubusercontent.com/36630876/152648878-56f560bb-fd23-48cf-a9d1-8e222ee103af.png)

**Comment by lord-marcel** [comment]
Hai ragione, lo correggo appena posso

**Comment by LuginiAndrea** [comment]
Soluzione al terzo punto: ipotizziamo di voler scrivere i punteggi ottenuti rispondendo come una stringa binaria di 7 elementi con valore 1 se corretta e 0 se scorretta -> contiamo tutti gli anagrammi della parola 1111100 -> 
7! / (5! * 2!)
